6starter.tex

```diff
@@ -210,38 +210,38 @@
 
  \hline
 %  \pagebreak
- 1 & Depletion basis: & \multirow{1}{1cm}[-0.25cm]{\parbox{12.5cm}{Selects the basis for the denominator when calculating degree of depletion in reproductive output (aka spawning biomass, aka B). The calculated values are reported to the SD report relative to a fraction, X, of a comparable quantity calculated in the benchmark section or elsewhere.}} \Tstrut\\
+ 1 & Depletion basis: & \multirow{1}{1cm}[-0.25cm]{\parbox{12.5cm}{Selects the basis for the denominator when calculating degree of depletion in reproductive output (a.k.a., \gls{sb}, a.k.a. B). The calculated values are reported to the \gls{sd} report relative to a fraction, X, of a comparable quantity calculated in the benchmark section or elsewhere.}} \Tstrut\\
```

Comment:
I don't see why "a.k.a. B" is there. B is biomass not spawning biomass.

Reply:
I too thought this but I wasn't sure if there was a reason for that the B was originally there. Will happily take it out unless @chantelwetzel-noaa knows why it's there and thinks it should be kept in.

Reply:
I suggest removing it.